build.gradle

```diff
@@ -74,29 +74,29 @@ subprojects {
         apply plugin: 'com.diffplug.spotless'
         spotless {
             format 'misc', {
-                target '**/*.md', '**/.gitignore'
```

Comment:
This is based on https://github.com/diffplug/spotless/issues/870#issuecomment-851726260, otherwise the build output will have lots of "[spotless depends out spotless](https://github.com/uber/motif/actions/runs/6499603365/job/17653264591#step:6:88)" stuff